spec/fixtures/files/xml/core/collectionobject1.xml

```diff
@@ -16,13 +16,13 @@
       <dateEarliestSingleCertainty/>
       <dateEarliestSingleQualifierUnit/>
       <dateDisplayDate>8-11-09</dateDisplayDate>
-      <dateLatestScalarValue>0009-08-12T00:00:00.000Z</dateLatestScalarValue>
+      <dateLatestScalarValue>2009-08-12T00:00:00.000Z</dateLatestScalarValue>
```

Comment:
There were several issues with the dates. Under the hood, the CollectionSpace application parses a 2-digit date literally (dropping any preceding zero). Date parsing in the mapper does not. Made an issue to consider whether to change this in the mapper: #59 

It looks like you had figured this out and changed the dates in the CollectionSpace record, but not re-saved the XML?

I just updated the XML to include the values that would be generated by the Mapper's date parser. That parser is tested heavily elsewhere in the Mapper specs, so what's important to test here is that the object returned by the date parser, whatever its values, is getting turned into XML properly.

Reply:
Yeah, I was getting the error with the dates and was making changes on it.